libfreerdp/core/gateway/rdg.c

```diff
@@ -1052,7 +1052,8 @@ static BOOL rdg_send_tunnel_request(rdpRdg* rdg)
 			return FALSE;
 		}
 
-		packetSize += 2 + (UINT32)(PAACookieLen + 1) * sizeof(WCHAR);
+		PAACookieLen += 1; /* include \0 */
```

Comment:
this looks wrong, if the content is `WCHAR` then the `\0` must be `sizeof(WCHAR)`

Reply:
sorry, looks like I need some coffee